docs/api/python/contrib/summary.md

```diff
@@ -0,0 +1,258 @@
+# Logging MXNet Data for Visualization in TensorBoard
```

Comment:
Is this file location the right place? I would have expected it to be at python/contrib/summary/README.md